src/main/java/com/list/LinkedList.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * File: LinkedList.java
+ * Author: Tarun Jain
+ * Date: 1 Feb, 2016
+ * -----------------------------------------
+ * LinkedList is a generic data structure to store data items
+ * of any type in list fashion.  
+ * 
```

Comment:
Add the API of linked List here, just like we have added in few other Data structures. For reference see graphs